internal/pkg/service/common/distribution/assigner_test.go

```diff
@@ -0,0 +1,59 @@
+package distribution_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/lafikl/consistent"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestConsistentHashLib tests the library behavior and shows how it should be used.
+func TestConsistentHashLib(t *testing.T) {
+	t.Parallel()
```

Comment:
A este som presunul tento test, ... ukazuje ako funguje pouziva libka (a testuje, ze je to tak aj v pripadnej novsej verzii).
Viac to patri k `assigner` ako ku `node` kodu.